Packages/com.unity.inputsystem/InputSystem/Plugins/Android/GetCustomCommand.cs

```diff
@@ -0,0 +1,45 @@
+#if UNITY_EDITOR || UNITY_ANDROID
+
+using System.Runtime.InteropServices;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.InputSystem.Utilities;
+
+namespace UnityEngine.InputSystem.Android
+{
+    /// <summary>
+    /// Command to get a custom value from the runtime
+    /// For example, on Android, this command can be used to query whether the back button
+    /// leaves the app or not.
+    /// </summary>
+    [StructLayout(LayoutKind.Explicit, Size = kSize)]
+    internal struct GetCustomCommand : IInputDeviceCommandInfo
+    {
+        public static FourCC Type { get { return new FourCC('G', 'C', 'C'); } }
+
+        internal const int kSize = InputDeviceCommand.kBaseCommandSize + sizeof(uint) * 2;
+
+        [FieldOffset(0)]
+        public InputDeviceCommand baseCommand;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize)]
+        public uint code;
+
+        [FieldOffset(InputDeviceCommand.kBaseCommandSize + sizeof(uint))]
+        public uint value;
```

Comment:
It is not clear to me what to expect from this field, also is "value" the most representative name for this field?

Reply:
renamed to payload https://github.com/Unity-Technologies/InputSystem/pull/1914/commits/30c57bc4277ce7695d89394cbb616824065f1d5c